querybuilder/rule_group_test.go

```diff
@@ -48,13 +80,18 @@ func TestRuleGroupEvaluate(t *testing.T) {
 		rules   string
 		dataset string
 		want    bool
+		err     string
```

Comment:
should be of type `error`. you can use `err.Error()` to get string representation